src/ViewItem.js

```diff
@@ -179,6 +189,27 @@ class ViewItem extends React.Component {
     }
   }
 
+  async componentDidUpdate(prevProps) {
+    if (prevProps.resources.items && this.props.resources.items.records[0] !== prevProps.resources.items.records[0]) {
```

Comment:
Please move this logic to a function with the meaningless name and call it inside `componentDidUpdate`.